forms-flow-web/src/components/Draft/Edit.js

```diff
@@ -0,0 +1,279 @@
+
+  if (isActive || isPublicStatusLoading) {
+    return (
+      <div data-testid="loading-view-component">
+        <Loading />
+      </div>
+    );
+  }
+
+  if (isFormSubmitted) {
+    return (
+      <div className="text-center pt-5">
+        <h1>{t("Thank you for your response.")}</h1>
+        <p>{t("saved successfully")}</p>
+      </div>
+    );
+  }
+
+  return (
+    <div className="container overflow-y-auto">
+      <div className="main-header">
+        <SubmissionError
+          modalOpen={props.submissionError.modalOpen}
+          message={props.submissionError.message}
+          onConfirm={props.onConfirm}
+        ></SubmissionError>
+        {isAuthenticated ? (
+          <Link title="go back" to={`${redirectUrl}form`}>
+            <i className="fa fa-chevron-left fa-lg" />
+          </Link>
+        ) : null}
+
+        {form.title ? (
+          <h3 className="ml-3">
+            <span className="task-head-details">
+              <i className="fa fa-wpforms" aria-hidden="true" /> &nbsp; Drafts /
+            </span>{" "}
+            {form.title}
+          </h3>
+        ) : (
+          ""
+        )}
+      </div>
+      <Errors errors={errors} />
+      <LoadingOverlay
+        active={isFormSubmissionLoading}
+        spinner
+        text={<Translation>{(t) => t("Loading...")}</Translation>}
+        className="col-12"
+      >
+        <div className="ml-4 mr-4">
+          <Form
+            form={form}
+            submission={submission.submission}
+            url={url}
+            options={{
+              ...options,
+              language: lang,
+              i18n: formio_resourceBundles,
+            }}
+            hideComponents={hideComponents}
+            onChange={(formData) => setDraftData(formData.data)}
+            onSubmit={(data) => {
+              setPoll(false);
+              onSubmit(data, form._id, isPublic);
+            }}
+            onCustomEvent={(evt) => onCustomEvent(evt, redirectUrl)}
+          />
+        </div>
+      </LoadingOverlay>
+    </div>
+  );
+});
+
+const executeAuthSideEffects = (dispatch, redirectUrl) => {
+  dispatch(push(`${redirectUrl}draft`));
+};
+
+// eslint-disable-next-line no-unused-vars
+const doProcessActions = (submission, ownProps) => {
+  return (dispatch, getState) => {
+    const state = getState();
+    let user = state.user.userDetail;
+    let form = state.form.form;
+    let isAuth = state.user.isAuthenticated;
+    dispatch(resetSubmissions("submission"));
+    const data = getProcessReq(form, submission._id, "new", user);
+    const tenantKey = state.tenants?.tenantId;
+    const redirectUrl = MULTITENANCY_ENABLED ? `/tenant/${tenantKey}/` : `/`;
+    let draft_id = state.draft.submission?.id;
+    let isDraftCreated = draft_id ? true : false;
+    const applicationCreateAPI = selectApplicationCreateAPI(
+      isAuth,
+      isDraftCreated,
+      DRAFT_ENABLED
+    );
+    dispatch(
+      // eslint-disable-next-line no-unused-vars
```

Comment:
We can remove from the arguments if not needed rather than having eslint suppression

Reply:
👍